.github/workflows/openapi_update.yaml

```
@@ -0,0 +1,29 @@
+name: Dispatch OpenAPI File Change event
+on:
+  push:
+    branches: 
+      - main
+    paths:
+      - "openapi/*.yaml"
```

It will be better to have exact name 
```suggestion
      - "openapi/openapi.yaml"
```